apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -207,7 +221,7 @@ class CensusForm extends Component {
     const { errors } = this.state;
     if (!errors.email && !errors.topics && !errors.frequency && !errors.school && !errors.nces && !errors.role && !errors.hoc && !errors.afterSchool && !errors.tenHours && !errors.twentyHours && !errors.country) {
       $.ajax({
-        url: "/forms/Census2017",
+        url: "/dashboardapi/v1/census/CensusYourSchool2017v4",
```

Comment:
should this match line 267 in apps/src/sites/hourofcode.com/pages/public/index.js or are they intentionally different?

Reply:
The different form versions map to different models with different validation. They all write to the same table (census_submission) but with a different value for 'type'. This also allows us to map responses back to which version of the form was used.